user/gui-and-headless-browsers.md

```diff
@@ -72,6 +72,14 @@ services:
 ```
 {: data-file=".travis.yml"}
 
+> Sometimes it may be necessary to explicitly set the DISPLAY on Travis CI Enterprise. You can achieve this with the following snippet:
+
+```yaml
+before_script:
+  - "export DISPLAY=:99.0"
```

Comment:
Do you know if it's specific to `dist: xenial`? If so, I would put the full recipe:

```yaml
dist: xenial
services:
  - xvfb

before_script:
  - "export DISPLAY=:99.0"
```

Reply:
No idea. @ssabrii please can you confirm? Thanks

Reply:
qguz9ZVRYsdtfzwnm6qX

Reply:
> Do you know if it's specific to `dist: xenial`? If so, I would put the full recipe:
> 
> ```yaml
> dist: xenial
> services:
>   - xvfb
> 
> before_script:
>   - "export DISPLAY=:99.0"
> ```

Reply:
> Do you know if it's specific to `dist: xenial`? If so, I would put the full recipe:
> 
> ```yaml
> dist: xenial
> services:
>   - xvfb
> 
> before_script:
>   - "export DISPLAY=:99.0"
> ```